app/elements/mdl-footer.html

```diff
@@ -1,201 +1,208 @@
-      -o-order: 2;
-      order: 2;
-    }
-    
-    .footer_area[small-desktop] .mdl-mini-footer--center-section {
-      width: auto;
-      -webkit-order: 0;
-      -moz-order: 0;
-      -ms-order: 0;
-      -o-order: 0;
-      order: 0;
-      width: 56%;
-    }
-    
-    .footer_area .mdl-mini-footer--right-section {
-      width: 100%;
-    }
-    
-    .footer_area[wide-layout] .mdl-mini-footer--right-section {
-      -webkit-order: 1;
-      -moz-order: 1;
-      -ms-order: 1;
-      -o-order: 1;
-      order: 1;
-      width: 117px;
-    }
-    
-    .footer_area[small-desktop] .mdl-mini-footer--right-section {
-      -webkit-order: 2;
-      -moz-order: 2;
-      -ms-order: 2;
-      -o-order: 2;
-      order: 2;
-    }
-    
-    .footer_area[wide-layout] .one_third {}
-    
-    .sponsor__link {
-      vertical-align: middle;
-      display: block;
-    }
-    
-    .sponsor__link:nth-child(1) {
-      width: 110px;
-    }
-    
-    .sponsor__link:nth-child(2) {
-      width: 190px;
-    }
-    
-    .sponsor__link:nth-child(3) {
-      width: 140px;
-    }
-    
-    .sponsor__link img {
-      width: 100%;
-      display: block;
-    }
-    
-    .footer_area[wide-layout] .sponsor__link {
-      text-decoration: none;
-      display: inline-block;
-      text-align: left;
-    }
-    
-    .footer_area[wide-layout] .sponsol__link {}
-    
-    .footer_area[wide-layout] .sponsor--margin {
-      margin: 0 2rem;
-    }
-    
-    .social {
-      margin-top: 16px;
-    }
-    
-    .social-list {
-      position: relative;
-      overflow: hidden;
-      padding-left: 13px;
-    }
-    
-    .social-list__item {
-      float: left;
-      margin: 0 3px;
-    }
-    
-    .social-list__link {
-      display: inline-block;
-    }
-    
-    .social-list__icon {
-      width: 45px;
-      display: inline-block;
-    }
-    
-    @media (max-width: 540px) {
-      .sponsor {
+
+      .footer_area[wide-layout] .mdl-mini-footer--right-section {
+        -webkit-order: 1;
+        -moz-order: 1;
+        -ms-order: 1;
+        -o-order: 1;
+        order: 1;
+        width: 117px;
+      }
+
+      .footer_area[small-desktop] .mdl-mini-footer--right-section {
+        -webkit-order: 2;
+        -moz-order: 2;
+        -ms-order: 2;
+        -o-order: 2;
+        order: 2;
+      }
+
+      .sponsor__link {
+        vertical-align: middle;
         display: block;
-        margin: 2rem 0;
       }
-      .sponsor--margin {
-        margin: 0;
+
+      .sponsor__link:nth-child(1) {
+        width: 110px;
+      }
+
+      .sponsor__link:nth-child(2) {
+        width: 190px;
+      }
+
+      .sponsor__link:nth-child(3) {
+        width: 140px;
+      }
+
+      .sponsor__link img {
+        width: 100%;
+        display: block;
+      }
+
+      .footer_area[wide-layout] .sponsor__link {
+        text-decoration: none;
+        display: inline-block;
+        text-align: left;
+      }
+
+      .footer_area[wide-layout] .sponsol__link {}
```

Comment:
I was just re-indenting styles in this file, but let me work on a quick fix for this.